kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewContent.vue

```diff
@@ -0,0 +1,252 @@
+          v-else
+          :text="addText$()"
+          :primary="false"
+          @click="addResource()"
+        />
+      </div>
+    </div>
+
+    <ResourceSelectionBreadcrumbs
+      v-if="ancestors.length"
+      :ancestors="[...ancestors, currentContentNode]"
+      :channelsLink="channelsLink"
+      :topicsLink="topicsLink"
+    />
+
+    <h2>
+      <KLabeledIcon :label="currentContentNode.kind">
+        <template #icon>
+          <LearningActivityIcon :kind="learningActivities" />
+        </template>
+        <template>
+          {{ currentContentNode.title }}
+        </template>
+      </KLabeledIcon>
+    </h2>
+
+    <ContentArea
+      :header="questionLabel(selectedQuestionIndex)"
+      :selectedQuestion="selectedQuestion"
+      :content="currentContentNode"
+      :isExercise="isExercise"
+    />
+
+    <SlotTruncator
+      v-if="description"
+      :maxHeight="75"
+      :showViewMore="true"
+    >
+      <!-- eslint-disable vue/no-v-html -->
+      <p
+        dir="auto"
+        v-html="description"
+      ></p>
+      <!-- eslint-enable -->
+    </SlotTruncator>
+
+    <HeaderTable class="license-detail-style">
+      <HeaderTableRow :keyText="coreString('suggestedTime')">
+        <template #value>
+          {{
+            currentContentNode.duration
+              ? getTime(currentContentNode.duration)
+              : notAvailableLabel$()
+          }}
+        </template>
+      </HeaderTableRow>
+
+      <HeaderTableRow :keyText="licenseDataHeader$()">
+        <template #value>
+          {{ licenseName }}
+        </template>
+      </HeaderTableRow>
+
+      <HeaderTableRow :keyText="copyrightHolderDataHeader$()">
+        <template #value>
+          {{ currentContentNode.license_owner }}
+        </template>
+      </HeaderTableRow>
+    </HeaderTable>
+  </div>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import { searchAndFilterStrings } from 'kolibri-common/strings/searchAndFilterStrings';
+  import { licenseLongName } from 'kolibri/uiText/licenses';
+  import markdownIt from 'markdown-it';
+  import { ContentNodeKinds } from 'kolibri/constants';
+  import LearningActivityIcon from 'kolibri-common/components/ResourceDisplayAndSearch/LearningActivityIcon.vue';
+  import SlotTruncator from 'kolibri-common/components/SlotTruncator';
+  import ContentArea from '../../../../../LessonSelectionContentPreviewPage/LessonContentPreview/ContentArea.vue';
+  import commonCoach from '../../../../../../common';
+  import { PageNames } from '../../../../../../../constants/index';
+  import ResourceSelectionBreadcrumbs from '../../../../../LessonResourceSelectionPage/SearchTools/ResourceSelectionBreadcrumbs.vue';
+
+  export default {
+    name: 'PreviewContent',
+    components: {
+      ContentArea,
+      SlotTruncator,
+      LearningActivityIcon,
+      ResourceSelectionBreadcrumbs,
+    },
+    mixins: [commonCoreStrings, commonCoach],
```

Comment:
Do we actually need this commonCoach mixin here? :eyes: What property of it are we using?

Reply:
This PR https://github.com/learningequality/kolibri/pull/13032 depends on what t luckily if we merge this we can rebase and polish up few things

Reply:
Yes we do need it here!  To render copy right , incense etc!